core/chains/evm/config/toml/defaults/Polygon_Zkevm_Mainnet.toml

```diff
@@ -4,6 +4,7 @@ NoNewHeadsThreshold = '6m'
 MinIncomingConfirmations = 1
 LogPollInterval = '30s'
 RPCBlockQueryDelay = 15
+RPCDefaultBatchSize = 100
```

Comment:
It might not be relevant since we lower the limit but it would be nice to confirm that Polygon zkEVM behaves the same way as L1 since it's a rollup and it might deviate from the original RPC client.

Reply:
Can't find the default but I found these values: https://github.com/0xPolygonHermez/zkevm-node/blob/9ef6f20949dfd4533fb64af2d1dfd012d8dbeba8/jsonrpc/config.go#L41-L45

Reply:
Found the default: https://github.com/0xPolygonHermez/zkevm-node/blob/54ef1c4ad600b396442ac2c2263594a80a10f45b/config/default.go#L90 seems to be 20.